### PR TITLE
Make execution time of `jump_to_unread_xxx()` more predictable

### PR DESCRIPTION
**Pull request by dennisschagt**

### CODE REVIEW

File: src/feedlistformaction.cpp [hunk 1]
```diff
@@ -692,25 +692,20 @@ void FeedListFormAction::goto_feed(const std::string& str)
...
-				utils::get_random_value(visible_feeds.size());
-			if (visible_feeds[pos].first->unread_item_count() > 0) {
-				feeds_list.set_position(pos);
-				feedpos = visible_feeds[pos].second;
-				break;
-			}
-		}
+	if (!unread_indexes.empty()) {
+		unsigned int selected = utils::get_random_value(unread_indexes.size());
+		unsigned int pos = unread_indexes[selected];
```
Comment (Minoru) [inline src/feedlistformaction.cpp:703]: Can these two be `const`?

File: src/itemlistformaction.cpp [hunk 2]
```diff
@@ -1152,24 +1152,19 @@ bool ItemListFormAction::jump_to_previous_unread_item(bool start_with_last)
...
-			unsigned int pos =
-				utils::get_random_value(visible_items.size());
-			if (visible_items[pos].first->unread()) {
-				items_list.set_position(pos);
-				break;
-			}
-		}
+	if (!unread_indexes.empty()) {
+		unsigned int selected = utils::get_random_value(unread_indexes.size());
+		unsigned int pos = unread_indexes[selected];
```
Comment (Minoru) [inline src/itemlistformaction.cpp:1163]: Can these two be `const`?

File: src/feedlistformaction.cpp (hunk 1 above)
Comment (dennisschagt) [reply src/feedlistformaction.cpp]: Jup, thanks.
Added const for both instances.

**Comment by dennisschagt** [comment]
The Travis build succeeded but it apparently didn't update the status on Github.
I will just merge it now.

**Pull request merged**